lib/practice_exercises.rb

```diff
@@ -1,13 +1,34 @@
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n) where n is the sorted array
+# Space Complexity: O(1)
 def remove_duplicates(list)
-  raise NotImplementedError, "Not implemented yet"
+  counter = 0
+  list.length.times do |num|
+    if list[counter] == list[counter + 1]
+      list.delete_at(counter + 1)
+    else  
+      counter += 1
+    end
+  end
+  return list
 end
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n) where n is the strings array
```

Comment:
If the strings are small yes, otherwise you can say O(n * m) where m is the min length of the strings.